Library/src/com/easyandroidanimations/library/FlipHorizontalToAnimation.java

```diff
@@ -37,9 +38,9 @@
 	 * @param view
 	 *            The view to be animated.
 	 */
-	public FlipHorizontalToAnimation(View view) {
+	public FlipHorizontalToAnimation(View fromView, View toView) {
 		this.view = view;
-		flipToView = null;
```

Comment:
set by the set method, we can introduce a custom exception for missing parameters later